azure-pipelines.yml

```
@@ -94,3 +99,18 @@ jobs:
           testResultsFiles: '$(testResultsFiles)'
           testRunTitle: 'Windows - $(name)'
           mergeTestResults: true
+  - job: GitHub_Release
+    displayName: Release to GitHub
+    dependsOn: 
+    - Windows
+    - Linux
+    condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/v1.'))  
+    steps:
+    - task: GitHubRelease@0
+      displayName: 'GitHub release (create)'
+      inputs:
+        gitHubConnection: 'github-zenithworks'
+        title: JupyterLab Release version $(Build.SourceBranchName) 
```

Maybe we should clarify this is the Python release version, not a JS release.

```suggestion
        title: JupyterLab Python version $(Build.SourceBranchName) 
```